Code/GraphMol/Depictor/Wrap/testDepictor.py

```diff
@@ -236,6 +237,149 @@ def testConstrainedCoords(self):
         rdDepictor.GenerateDepictionMatching3DStructure(xp0_lig_2d, xp0_lig, -1, xp0_ref)
 
 
+    def testGenerate2DDepictionRefPatternAtomMap(self):
+        indazoleMolblock = """
+     RDKit          2D
+
+  9 10  0  0  0  0  0  0  0  0999 V2000
+   -6.0878    2.4335    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -7.3867    1.6835    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -7.3867    0.1833    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -6.0878   -0.5666    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -4.7887    0.1833    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -4.7887    1.6835    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -3.4897   -0.5664    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.1906    1.6833    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.1906    0.1835    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  2  0
+  2  3  1  0
+  3  4  2  0
+  4  5  1  0
+  5  6  2  0
+  6  1  1  0
+  8  9  2  0
+  6  8  1  0
+  7  9  1  0
+  7  5  1  0
+M  END)RES"""
```

Comment:
I guess the `)RES` here is a copy/paste wart?

Reply:
Oh, yes. Fixed. (Great that it didn't choke the `MolBlock` parser!).